2019Labs/RHELSecurityLab/documentation/lab1_OpenSCAP.adoc

```diff
@@ -66,57 +66,54 @@ Notice that this command outputs the OpenSCAP version and versions of supported
 +
  [root@openscap]# rpm -ql scap-security-guide
  ...
- /usr/share/xml/scap/ssg/content/ssg-rhel7-cpe-dictionary.xml
- /usr/share/xml/scap/ssg/content/ssg-rhel7-cpe-oval.xml
- /usr/share/xml/scap/ssg/content/ssg-rhel7-ds.xml
- /usr/share/xml/scap/ssg/content/ssg-rhel7-oval.xml
- /usr/share/xml/scap/ssg/content/ssg-rhel7-xccdf.xml
+ /usr/share/xml/scap/ssg/content/ssg-rhel8-cpe-dictionary.xml
+ /usr/share/xml/scap/ssg/content/ssg-rhel8-cpe-oval.xml
+ /usr/share/xml/scap/ssg/content/ssg-rhel8-ds.xml
+ /usr/share/xml/scap/ssg/content/ssg-rhel8-ocil.xml
+ /usr/share/xml/scap/ssg/content/ssg-rhel8-oval.xml
+ /usr/share/xml/scap/ssg/content/ssg-rhel8-xccdf.xml
  ...
 
-. Notice that the provided content in `scap-security-guide` covers a wide range of security baselines such as PCI-DSS, STIG, C2S, Common, CJIS, and others. Also notice that Ansible remediation playbooks are included for these various security baselines as well in the provided content in the `scap-security-guide` .
+. Notice that the provided content in `scap-security-guide` covers a wide range of security baselines such as PCI-DSS, STIG, C2S, Common, CJIS, and others. Also notice that Ansible remediation playbooks are included for these various security baselines as well in the provided content in the `scap-security-guide`. 
```

Comment:
IIRC our RHEL8 content is limited, although this statement doesn't mention RHEL8 explicitly, I would somehow tune it down, in a similar manner as you did below.